### ISSUE TITLE
broken image link in recurrent_quickdraw.md

### ISSUE THREAD

**Issue by harmindergit** [tensorflow/tensorflow#30127]
Description of issue: The image link for quickdraw_model.png is broken in document [recurrent_quickdraw.md](https://github.com/tensorflow/docs/blob/master/site/en/tutorials/sequences/recurrent_quickdraw.md) 

The current source url for the image points to non existent image location.

Url: [recurrent_quickdraw.md](https://github.com/tensorflow/docs/blob/master/site/en/tutorials/sequences/recurrent_quickdraw.md)

**Comment by ymodak** [comment]
Unable to locate the broken link in the markdown file referenced in the issue. Can you please try to elaborate? Thanks!

**Comment by harmindergit** [comment]
the link to the image (Line 16)

![RNN model structure](../../images/quickdraw_model.png) is not correct.

it is below this paragraph
"In this tutorial we'll show how to build an RNN-based recognizer for this problem. The model will use a combination of convolutional layers, LSTM layers, and a softmax output layer to classify the drawings:"

**Comment by lamberta** [comment]
Thanks. Fixed markdown for github
